public/scripts/char-data.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @typedef {object} WorldInfoEntry
```

Comment:
This is neither an entirely correct nor exhaustive description of a WI entry. 
You can find a template for the entry with all fields here: https://github.com/SillyTavern/SillyTavern/blob/release/public/scripts/world-info.js#L1716

Reply:
I renamed it to something like `v2DataWorldInfoEntry`, is that ok?
https://github.com/SillyTavern/SillyTavern/commit/2f960f8b4fa261b841be418422aa27ebaba6b262
Since the original project was only dealing with data formats in png so yes This description only applies to the v2 std.